users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -791,6 +791,128 @@ def run_scf_baseline():
     return report
 
 
+def run_scf_frozen_features():
+    gs.ALIAS_AND_OUTPUT_SUBDIR = "experiments/switchboard/ctc/feat/"
+
+    (
+        returnn_datasets,
+        rasr_loss_corpus_path,
+        rasr_loss_corpus_segments,
+        rasr_loss_lexicon_path,
+        dev_corpora,
+    ) = get_datasets()
+    returnn_args = {
+        "batch_size": 5000,
+        "rasr_binary_path": RASR_BINARY_PATH,
+        "rasr_loss_corpus_path": rasr_loss_corpus_path,
+        "rasr_loss_corpus_segments": rasr_loss_corpus_segments,
+        "rasr_loss_lexicon_path": rasr_loss_lexicon_path,
+        "datasets": returnn_datasets,
+        "conformer_type": "wei",
+        "specaug_old": {"max_feature": 15},
+    }
+    feature_args = {
+        "class": "ScfNetwork",
+        "size_tf": 256 // 2,
+        "stride_tf": 10 // 2,
+        "preemphasis": 0.97,
+        "wave_norm": True,
+    }
+
+    nn_args, report_args_collection = get_nn_args_baseline(
+        nn_base_args={
+            "scf_baseline": dict(
+                returnn_args={
+                    "staged_opts": {
+                        1: "freeze_features",
+                        16: "unfreeze_features",
+                    },
+                    "extra_args": {
+                        "accum_grad_multiple_step": 2,
+                        "conv_pad_seq_len_to_power": 1.5,
+                        "preload_from_files": {
+                            "existing-model": {
+                                "filename": "/u/maximilian.kannen/setups/20230406_feat/work/i6_core/returnn/training/ReturnnTrainingJob.xUBplWqSsRII/output/models/epoch.016.index",
```

Comment:
You should never use absolute paths as strings in a setup like this. The clean solution would be to get the checkpoint from a previous training. In case this is just experimental for you to test this, I'd rather wait for you to check the results and only clean and merge this if it has shown to be helpful.